### ISSUE TITLE
Analytics Reporting Request for Qaysar VANotify

### ISSUE THREAD

**Issue by QaysarA** [department-of-veterans-affairs/va.gov-team#12698]
# Metrics Reporting Request

## What this form is for

Use this template to request one-off performance, usability, or outcomes va.gov metrics that is difficult to access or that needs analytics interpretation.

Insights will usually provide data reports within the current sprint, but if there is time-sensitivity, please note them in "additional comments."

## Instructions

Please fill out the following sections:
- [X] Description
- [X] Timeframe for Analytics Requested
- [x] Analytics to Track Down
- [X] Which Product Questions are you trying to answer?
- [X] Additional Comments

## Description
Who is this request for?

- Qaysar Ahmad

    _Send Slack message to DSVA Slack Channel, `#VFS-Platform-Support`, that includes: email address(es) for user(s) requesting data and link to this Github issue._
    
- What team/project is this work for?

  - VANotify (Notifications for internal VA teams) 
    

## Timeframe for Analytics Requested
Looking at data from the previous year - current (example August 2019 - August 2020)

## Which Product Questions are you trying to answer?

- Total number of applications/forms that are successfully submitted on VA.gov (approx. monthly volume, and from the previous year) 
- Details and breakdown on specific forms/applications (example - The application/form 22-1995 was successfully completed by Veterans on VA.gov an average of 22K/month and 250K last year total, The application/form 2346A was successfully completed by Veterans on VA.gov an average of 10 times month and 106 last year total Aug 2019 -2020) 

## Other Helpful Questions to think through
- Who is your audience?
Internal team and a single request for discovery/decision making 

- Why do you need this data?
In discovery, help with data driven decision on notifications volume and approach based on the total number of applications submitted on VA.gov 

- What are you hoping to find?
Total number of applications/forms submitted on VA.gov, the ability to look into details of each specific application/form (example volume or yearly trends/spikes for - Form 10-10EZ, Form 21P-530) 

- What is your hypothesis that you're hoping the data will support?
Help with discovery and prioritize notifications for successfully completed applications/forms on VA.gov

- What area of the VA are you hoping to assess?
Any application/form submitted on VA.gov, with the focus to prioritize on high volume forms/applications that are successfully submitted   

- How frequently do you need this data?
Single request/once  

## Additional Comments
Joanne was able to help with some of the forms/applications submitted in Pensions, Burials, Healthcare, Education Benefits 

> Please leave the following blank

## Acceptance Criteria
- [ ] The data has been identified and documented in this ticket - items that we don't have access to are noted with _cannot be accessed_
- [ ] The metrics are shared back with users.

## Definition of Done
- [ ] All appropriate issue tagging is completed
- [ ] All AC completed

**Comment by bmcgrady-ep** [comment]
@QaysarA - Thank you for the clarification. I will let you know when this request is fulfilled.

**Comment by bmcgrady-ep** [comment]
@QaysarA - Here is a [custom report ](https://analytics.google.com/analytics/web/template?uid=ekhoyxZbRRqi5bVBnC4H4Q)that I created. When opening the custom report, just select All VA.gov View [Production]. Due to Google Analytics' reporting capabilities, I could not get the monthly average for the successful form submissions, but I did download the data into Excel and got the monthly average. 

Here is the Excel spreadsheet with the data. Let me know if you have any more questions.
[Form Successful Submissions.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/5126399/Form.Successful.Submissions.xlsx)
